.github/workflows/ci.yml

```diff
@@ -132,7 +136,9 @@ jobs:
       - uses: codecov/codecov-action@v4.1.0
         with:
           directory: ./.tmp
-          fail_ci_if_error: true
+          #fail_ci_if_error: true
+          # PR from dependabot causes an error, so comment this out until the problem is resolved.
+          # See https://github.com/codecov/codecov-action/issues/1274
```

Comment:
Let's remove it completely. `fail_ci_if_error` defaults to false and we should keep it that way. Having reports uploaded each time is not that important.